src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+foreach(PROG test_elements)
+  file(GLOB ${PROG}_SOURCES ${PROG}*.cc)
+  add_executable(unit_${PROG} ${${PROG}_SOURCES})
+  target_link_libraries(unit_${PROG} votca_tools)
```

Comment:
You can put these three lines inside the `if(ENABLE_TESTING)` block, because there is no point in building the executable without testing.